docs/source/counts.rst

```diff
@@ -8,7 +8,7 @@ Counting over an Integer Range
 
 .. function:: counts(x, a:b[, wv])
 
-    Count the number of times (or total weights if a weight vector ``wv`` is given) values in ``a:b`` appear in array ``x``. Here, the optional argument ``wv`` should be a weight vector of type ``WeightVec`` (see :ref:`weightvec`).
+    Count the number of times (or total weights if a weight vector ``wv`` is given) values in ``a:b`` appear in array ``x``. Here, the optional argument ``wv`` should be a weight vector of type ``AbstractWeights`` (see :ref:`weightvec`).
```

Comment:
Yeah, I didn't rename the file. I assume that's what the `:ref:` is for (I really don't know restructured text though)?